lstchain/scripts/lstchain_dvr_pixselector.py

```diff
@@ -0,0 +1,516 @@
+            continue
+
+        cosmic_mask   = event_type_data == EventType.SUBARRAY.value  # showers
+        pedestal_mask = event_type_data == EventType.SKY_PEDESTAL.value
+
+        data_images = read_table(dl1_file, "/dl1/event/telescope/image/LST_LSTCam")
+
+        data_calib = read_table(dl1_file, "/dl1/event/telescope/monitoring/calibration")
+        # data_calib['unusable_pixels'] , indices: (Gain  Calib_id  Pixel)
+
+        # Get the "unusable" flags from the pedcal file:
+        unusable_HG = data_calib['unusable_pixels'][0][0]
+        unusable_LG = data_calib['unusable_pixels'][0][1]
+
+        #  Keep pixels that could not be properly calibrated: save them always,
+        #  in case their calibration can be later fixed:
+        keep_always_mask = (unusable_HG | unusable_LG)
+        summary_info.number_of_always_saved_pixels = keep_always_mask.sum()
+
+        if summary_info.number_of_always_saved_pixels > 0:
+            print(f'Pixels that will always be saved ('
+                  f'{summary_info.number_of_always_saved_pixels}):')
+            for pixindex in np.where(keep_always_mask)[0]:
+                print(pixindex)
+        else:
+            print('Pixels that will always be saved: None')
+
+        charges_data = data_images['image']
+        # times_data = data_images['peak_time'] # not used now
+        image_mask = data_images['image_mask']
+
+        print("Original standard cleaning, pixel survival probabilities:")
+        print('  Minimum: ', np.round(np.mean(image_mask, axis=0).min(), 5))
+        print('  Maximum: ', np.round(np.mean(image_mask, axis=0).max(), 5))
+        print('  Mean: ', np.round(np.mean(image_mask, axis=0).mean(), 5))
+
+        charges_cosmics = charges_data[cosmic_mask]
+        if not write_pixel_masks:
+            # Calculate an adequate data volume reduction pixel threshold for
+            # this subrun:
+            min_charge_for_certain_selection = find_DVR_threshold(charges_cosmics,
+                                                                  max_pix_survival_fraction,
+                                                                  args.picture_threshold,
+                                                                  camera_geom)
+        else:
+            runmask = dvr_settings['run_id'] == run_id
+            meanq = dvr_settings['min_charge_for_certain_selection'][runmask].mean()
+            min_charge_for_certain_selection  = np.floor(meanq)
+            # we do not have to calculate it from scratch, we get the
+            # value (same for all subruns) from the DVR_settings_LST*.h5 file,
+            # just averaging the subrun-wise values and taking the closest
+            # (lower) integer
+
+        summary_info.min_charge_for_certain_selection = min_charge_for_certain_selection
+
+        # Cuts to identify promising muon ring candidates:
+        mucan_event_list = np.where(cosmic_mask &
```

Comment:
these hardcoded values should be defined elsewhere (at the beginning of the script, maybe?) with a comment on why they are chosen